float32 x
float32 y
float32 z
---
user_pkg/Voxel voxel
bool success